### PR TITLE
feat: add docker image to hosting

### PR DESCRIPTION
**Pull request by shyim**

### CODE REVIEW

File: guides/hosting/installation-updates/docker.md [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+
+::: info
+You can create a Project with a specific Shopware version by specifiying the version like: `composer create-project shopware/production:6.6.7.0 <folder>`
+:::
+
+
+```bash
+composer create-project shopware/production <folder>
+cd <folder>
+composer require shopware/docker
```
Comment (renaudhager) [inline guides/hosting/installation-updates/docker.md:21]: ` shopware/k8s-meta` is missing ^^

File: guides/hosting/installation-updates/docker.md [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+  build:
+    context: .
+  environment:
+    APP_ENV: prod
+    DATABASE_URL: 'mysql://shopware:shopware@database/shopware'
+    DATABASE_HOST: 'database'
+    APP_URL: 'http://localhost:8000'
+    APP_SECRET: 'test'
+    PHP_SESSION_HANDLER: redis
+    PHP_SESSION_SAVE_PATH: 'tcp://cache:6379/1'
```
Comment (bojanrajh) [inline guides/hosting/installation-updates/docker.md:185]: If we provided the database service (which is far from being production-ready or highly available), shouldn't we also provide example for the cache service (Redis)? Maybe define both in a separate stack and add network config to the `x-environment: $shopware`? This does make the config more complicated, but also draws the line between shopware- and external-provided services, makes it easier to run multiple web/app stacks with one shared stack running independent services (maria, redis, adminer, ...).

Comment (shyim) [reply guides/hosting/installation-updates/docker.md:185]: I wanted to visualize how a deployment could look like and how everything works with each other. for production I would just document later on a new page the kubernetes operator https://github.com/shopware/shopware-operator

Comment (shyim) [reply guides/hosting/installation-updates/docker.md:185]: Would you stay on a docker compose, or I draw something in draw.io 🤔

File: guides/hosting/installation-updates/docker.md (hunk 1 above)
Comment (TrayserCassa) [reply guides/hosting/installation-updates/docker.md:21]: No this is only for kubernetes cluster with the operator interesting

File: guides/hosting/installation-updates/docker.md [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+| images                 | /var/www/html/public/media     |
+| image thumbnails       | /var/www/html/public/thumbnail |
+| generated sitemap      | /var/www/html/public/sitemap   |
+
+
+Shopware logs by default to `var/log`, but when `shopware/docker` Composer package is installed, we change it to stdout. This means you can use `docker logs` to see the logs or use logging driver to forward the logs to a logging service.
+
+## Ideal Setup
+
+The ideal setup requires an external storage provider like S3. In that way you can don't need any mounts and can scale the instances without any problems.
```
Comment (TrayserCassa) [inline guides/hosting/installation-updates/docker.md:163]: can don't = can't :sweat_smile:

File: guides/hosting/installation-updates/docker.md (hunk 2 above)
Comment (TrayserCassa) [reply guides/hosting/installation-updates/docker.md:185]: I like the idea to have a working running docker-compose.yaml file, so I would stick with it. A side node that other services like valkey could also be used, is maybe also worth it.

File: guides/hosting/configurations/shopware/environment-variables.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+---
+
+# Environment Variables
+
+This page lists all environment variables that can be used to configure Shopware.
+
+| Variable                             | Default Value    | Description                                                                              |
+|--------------------------------------|------------------|------------------------------------------------------------------------------------------|
+| APP_ENV                              | prod             | Environment                                                                              |
```
Comment (mitelg) [inline guides/hosting/configurations/shopware/environment-variables.md:14]: don't forget `APP_BUILD_DIR` and `APP_LOG_DIR` :wink:

**Comment by mitelg** [comment]
nice work!

**Pull request merged**